cms/toolbar/items.py

```diff
@@ -222,11 +224,16 @@ def add_break(self, identifier=None, position=None):
         return item
 
     def get_items(self):
-        return self.items
```

Comment:
Because when a parent is disabled, all items should *appear* disabled, even if they're not.